OpenRA.Mods.Common/Effects/Missile.cs

```diff
@@ -757,8 +757,10 @@ public void Tick(World world)
 			// Check if target position should be updated (actor visible & locked on)
 			var newTarPos = targetPosition;
 			if (args.GuidedTarget.IsValidFor(args.SourceActor) && lockOn)
-				newTarPos = args.GuidedTarget.CenterPosition
-					+ new WVec(WDist.Zero, WDist.Zero, info.AirburstAltitude);
+			{
+				var airburstVec = new WVec(WDist.Zero, WDist.Zero, info.AirburstAltitude);
+				newTarPos = (!args.IgnoreTargetPointOffset ? args.GuidedTarget.TargetingCenterPosition : args.GuidedTarget.CenterPosition) + airburstVec;
```

Comment:
You can simplify that ternary.
`args.IgnoreTargetPointOffset ? args.GuidedTarget.CenterPosition : args.GuidedTarget.TargetingCenterPosition`